lib/seafoam/passes/truffle_translators/default.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "set"
```

Comment:
Not used?

Reply:
Bad refactoring. It should be in the _translators.rb_ file. I'll delete it.